examples/earthquakes.jGIS

```diff
@@ -1,117 +1,73 @@
       "parameters": {
         "color": {
-          "circle-fill-color": [
-            "case",
-            [
-              "==",
-              [
-                "get",
-                "tsunami"
-              ],
-              0.0
-            ],
-            [
-              125.0,
-              0.0,
-              179.0,
-              1.0
-            ],
-            [
-              "==",
-              [
-                "get",
-                "tsunami"
-              ],
-              1.0
-            ],
-            [
-              147.0,
-              255.0,
-              0.0,
-              1.0
-            ],
-            [
-              0.0,
-              0.0,
-              0.0,
-              0.0
-            ]
-          ],
-          "circle-radius": [
-            "interpolate",
-            [
-              "linear"
-            ],
-            [
-              "get",
-              "mag"
-            ],
-            1.0,
-            1.0,
-            2.0,
-            2.0,
-            3.0,
-            3.0,
-            4.0,
-            4.0,
-            5.0,
-            5.0,
-            6.0,
-            6.0
-          ],
-          "circle-stroke-color": "#3399CC",
+          "circle-fill-color": "#f66151",
+          "circle-radius": 5.0,
+          "circle-stroke-color": "#62a0ea",
           "circle-stroke-line-cap": "round",
           "circle-stroke-line-join": "round",
           "circle-stroke-width": 1.25
         },
         "opacity": 1.0,
-        "source": "dc048820-75cd-4b8d-a1fb-91642901cd82",
+        "source": "348d85fa-3a71-447f-8a64-e283ec47cc7c",
         "symbologyState": {
-          "colorRamp": "cool",
-          "mode": "",
-          "nClasses": "",
-          "renderType": "Categorized",
-          "value": "tsunami"
+          "renderType": "Single Symbol"
         },
         "type": "circle"
       },
       "type": "VectorLayer",
       "visible": true
+    },
+    "8de7c2c0-6024-4716-b542-031a89fb87f9": {
+      "name": "OpenStreetMap.Mapnik Layer",
+      "parameters": {
+        "source": "b2ea427a-a51b-43ad-ae72-02cd900736d5"
+      },
+      "type": "RasterLayer",
+      "visible": true
     }
   },
   "metadata": {},
   "options": {
     "bearing": 0.0,
     "extent": [
-      -14291047.530673811,
-      -3536164.7121253638,
-      -9426274.876637986,
-      9088825.15152122
+      -8743782.33482077,
+      -9086918.06770886,
+      3974641.244421752,
+      19283053.908469707
     ],
-    "latitude": 24.187972965810673,
-    "longitude": -106.52816608439294,
+    "latitude": 41.57863106624126,
+    "longitude": -21.420961668139388,
     "pitch": 0.0,
     "projection": "EPSG:3857",
-    "zoom": 3.8783091860507373
+    "zoom": 2.6670105136699993
   },
   "sources": {
-    "a7ed9785-8797-4d6d-a6a9-062ce78ba7ba": {
+    "348d85fa-3a71-447f-8a64-e283ec47cc7c": {
+      "name": "Custom GeoJSON Layer Source",
+      "parameters": {
+        "path": "../../examples/eq.json"
```

Comment:
I'm curious to know why the path changed, we may have a bug somewhere around the file selector. This path change makes it fail when I try to load that file in JupyterLite, it should stay:

```suggestion
        "path": "eq.json"
```